Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/profile_page.tsx

```diff
@@ -35,6 +35,9 @@ interface Properties {
   /** Whether the form can be edited. */
   readonly?: boolean;
 
+  /** Whether the password can be edited. */
```

Comment:
Prefer to use a common adjective like read-only to indicate whether a field can be modified, ie. ```isPasswordReadOnly```.